content/en/apps/guides/data/analytics/production.md

```diff
@@ -9,6 +9,76 @@ relatedContent: >
   core/overview/cht-sync
 ---
 
-{{% pageinfo %}}
-**This is under active development.** The page will be updated accordingly once it's validated that the current approach is suited for being used in production. For the time being, you can try out and test CHT Sync [locally]({{< relref "setup" >}}).
-{{% /pageinfo %}}
+We recommend running cht-sync in production using Kubernetes. This guide will walk you through setting up a production deployment of CHT Sync with the CHT using Kubernetes.
+
+## Pre-requisites:
+- A Kubernetes cluster: You can use a managed Kubernetes service like Google Kubernetes Engine (GKE), Amazon Elastic Kubernetes Service (EKS), or Azure Kubernetes Service (AKS), or you can set up a cluster using a tool like Minikube.
+- kubectl: The Kubernetes command-line tool. You can install it using the instructions [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+- Helm: The Kubernetes package manager. You can install it using the instructions [here](https://helm.sh/docs/intro/install/).
+
+## Setup
+- Copy the values in `values.yaml.template` file to a new file named `values.yaml`.
+- If you require a Postgres database to be set up for you, you can use the `postgresql.enabled` flag in the `values.yaml` file. If you already have a Postgres database set up, you can set the `postgresql.enabled` flag to `false`.
```

Comment:
```suggestion
- If you require a Postgres database to be set up in the cluster, you can use the `postgresl.enabled` flag in the `values.yaml` file. If you already have a Postgres database outside the cluster, you can set the `postgres.enabled` flag to `false`.
- if outside the cluster, specify `host` and `port` in this section
- in either case, specify username, password, database name, schema, and table name
- schema can be used to separate cht models from any other data that may already be in the database
- table is the name of the table that couch2pg will write couch documents to, and the source table for dbt models. It is recommended to leave this as `couchdb`
```

change postgresql to postgres
the other values are required even if enabled = true, enabled is really whether a postgres will be created inside the cluster or if it already exists outside
and if it oustide, need host and port
if it's too confusing, we could change the names to be something more explicit in the helm chart itself